packages/p2p/src/components/modal-manager/modals/disclaimer-modal/disclaimer-modal.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Button, Checkbox, Icon, Modal, Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+const DisclaimerModal = () => {
+    const [is_checked, setIsChecked] = React.useState<boolean>(false);
```

Comment:
```suggestion
    const [is_checked, setIsChecked] = React.useState(false);
```

Don't need to define type for it
It automatically infer the boolean type